templates/index.html

```diff
@@ -17,505 +19,653 @@
+          <p class="p-heading--2"><a href="https://canonical.com/careers/career-explorer">Browse careers&nbsp;›</a></p>
+        </div>
       </div>
-      <p>We rate top for Linux security. We run more hosts, more workloads and more devices than anybody else, because we strive to do everything insightfully, properly, fairly, and openly.</p>
-      <blockquote class="p-pull-quote--inverted">
-        <p class="p-pull-quote__quote">These are exciting challenges and we measure ourselves against the world leaders. It’s tough, but satisfying</p>
-      </blockquote>
-      <hr class="u-hide u-show--small"/>
     </div>
-    <div class="col-4">
-      <div class="p-heading-highlight--bottom">
-        <h3 class="p-heading-highlight--bottom__item">Enterprise focused</h3>
+  </div>
+</section>
+<section class="p-section">
+  <div class="row">
+    <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+      <hr class="p-rule is-dark is-bold">
+      <div class="row">
+        <div class="col-9">
+          <div class="p-block">
+            <h2 class="p-text--small-caps">Why open source?</h2>
+          </div>
+          <p class="p-heading--2">The future is already here, as open source. <br class="u-hide--medium u-hide--small">We deliver it to the world.</p>
+          <p class="p-heading--2"><a href="https://canonical.com/projects">How we contribute&nbsp;›</a></p>
+        </div>
       </div>
-      <p>The world is moving to Ubuntu &ndash; open source that's faster, cheaper and better. Empower engineers, secure precious data, share knowledge and lead the change.</p>
-      <blockquote class="p-pull-quote--inverted">
-        <p class="p-pull-quote__quote">My customers are transforming their operations around cloud, and I get to show them that future</p>
-      </blockquote>
     </div>
   </div>
-  <div class="row u-hide--small">
-    <hr />
-  </div>
+</section>
+<section class="p-section">
   <div class="row">
-    <div class="col-12">
-      <p><a href="/careers/career-explorer" class="p-button--outline">Explore a career at Canonical</a></p>
+    <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+      <hr class="p-rule is-dark is-bold">
+      <div class="row">
+        <div class="col-9">
+          <div class="p-block">
+            <h2 class="p-text--small-caps">What’s new</h2>
+          </div>
+          <p class="p-heading--2">See our mission in action. <br class="u-hide--medium u-hide--small">Read the latest news from Canonical.</p>
+          <p class="p-heading--2"><a href="/press-centre">Press centre&nbsp;›</a></p>
+        </div>
+      </div>
     </div>
   </div>
-</div>
+</section>
 
 <script>
-  var animateHTML = function() {
-    var elementsToAnimate;
-    var windowHeight;
-    function init() {
-      elementsToAnimate = document.querySelectorAll('[data-animation]');
-      windowHeight = window.innerHeight;
-      addEventHandlers();
-      checkPosition();
+    (function () {
```

Comment:
Is this tab behavior meant only for the homepage?

Reply:
this is copied from the vanilla example, not sure if it is needed elsehwere. What should I do?

Reply:
What happens if you don't add any JS? Does the navigation stop working? If so I think something broke elsewhere